packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/close/acceptProposal.ts

```diff
@@ -0,0 +1,87 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TransactionResponse } from '@chainlink/gauntlet-terra'
+import { CATEGORIES } from '../../../../lib/constants'
+import { AbstractInstruction, instructionToCommand, BeforeExecute } from '../../../abstract/executionWrapper'
+import { EMPTY_CONFIG } from './proposeOffchainConfig'
+
+type CommandInput = {
+  proposalId: string
+  digest: string
+}
+
+type ContractInput = {
+  id: string
+  digest: string
+}
+
+const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  if (flags.input) return flags.input as CommandInput
+  return {
+    proposalId: flags.proposalId || flags.configProposal, // --configProposal alias requested by eng ops
```

Comment:
Let's allow the same params names as contract expects:

```suggestion
    proposalId: flags.id || flags.proposalId || flags.configProposal, // --configProposal alias requested by eng ops
```